### ISSUE TITLE
Complete uninstall?

### ISSUE THREAD

**Issue by RealFX-Code** [NsCDE/NsCDE#130]
So I installed nscde, and now I want to remove it. I installed it on ubuntu 22.04.1 using the .deb package from the releases section... Now when I run any sysinfo application, E.G. hyfetch,
it says that I have NsCDE icons installed. How do I do a FULL removal of NsCDE?
![2022-11-17_16-06](https://user-images.githubusercontent.com/25106569/202482535-e64162f3-1e0e-4ef2-b3d0-b01d09d94962.png)

**Comment by NsCDE** [comment]
The "apt purge nscde" should do the job complitely.

You probably need to run qt5ct and similar gtk or some desktop tool to change icon and widget themes in ~/.config/qt5ct/qt5ct.conf and ~/.gtk2rc and ~/.config/gtk-3.0/settings.ini.

**Comment by RealFX-Code** [comment]
> The "apt purge nscde" should do the job complitely.
> 
> You probably need to run qt5ct and similar gtk or some desktop tool to change icon and widget themes in ~/.config/qt5ct/qt5ct.conf and ~/.gtk2rc and ~/.config/gtk-3.0/settings.ini.

I run `apt purge nscde` and GKrell System Monitor, KSH, the QT icon scheme and the DE beneath NsCDE was still remaining.

Now uninstalling those is easy enough, but I have no clue how to remove the QT icons...

**Comment by NsCDE** [comment]
Ok @RealFX-Code, can you chare with me how did you purged information for Qt5 icons, so I can put it in uninstall of the package or autoconf if appropriate? Thanks.

**Comment by RealFX-Code** [comment]
> Ok @RealFX-Code, can you chare with me how did you purged information for Qt5 icons, so I can put it in uninstall of the package or autoconf if appropriate? Thanks.

In the QT Config file, `~/.config/qt5ct/qt5ct.conf` There was an entry for icons, I simply removed the NsCDE icons from that list, and so it was gone from E.G. Neofetch and other programs that get information about icons and stuff. What I did, Was replace NsCDE with GTK2, so that it says the icon set that I've chosen in Gnome Tweaks, instead of some icons that I chose in that config file.

Image that shows NsCDE Icons in the QT5 config.
![Screenshot from 2022-12-14 21-40-00](https://user-images.githubusercontent.com/25106569/207709322-e8771899-65a7-4586-8e8a-cb3d1f307d2d.png)
